### ISSUE TITLE
formatter.relativeTime outputs wrong relative time

### ISSUE THREAD

**Issue by alex289** [amannn/next-intl#757]
### Description

I just noticed on my blog the relative time is wrong.

In my case I have this date on my post: "2022-07-11"
The relativeTime output is: "Last year"

This is my i18n.ts:

```ts
import { type AbstractIntlMessages } from 'next-intl';
import { getRequestConfig } from 'next-intl/server';

export default getRequestConfig(async ({ locale }) => ({
  messages: (
    (await import(`./messages/${locale}.json`)) as {
      default: AbstractIntlMessages;
    }
  ).default,
  timeZone: 'Europe/Berlin',
  now: new Date(),
}));
```

My component looks like this
```tsx
export default async function Blog({
  params,
}: {
  params: { slug: string; locale: string };
}) {
  unstable_setRequestLocale(params.locale);

  const now = await getNow({ locale: params.locale });
  const formatter = await getFormatter({ locale: params.locale });

return (
...
<p className="ml-2 text-sm text-gray-700 dark:text-[#c2c2c2]">
            {formatter.dateTime(new Date(post.publishedAt), {
              year: 'numeric',
              month: 'short',
              day: 'numeric',
            })}{' '}
            ({formatter.relativeTime(new Date(post.publishedAt), now)})
          </p>
...)}
```

You can preview the page (here)[https://alexanderkonietzko.vercel.app/en/blog/vercel-deployment]

Did I make a mistake here?

### Mandatory reproduction URL (CodeSandbox or GitHub repository)

https://github.com/alex289/Portfolio

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Go to the post page
3. See the wrong relative time

### Expected behaviour

Since "2022-07-11" is more than one year ago I expected something like "2 years ago"

**Comment by alex289** [comment]
On posts which are older than "2022-07-11" the correct relative time is shown

**Comment by amannn** [comment]
Thanks for the hint! This happened due to rounding because the current date and 2022-07-11 are just a bit less than 1.5 years apart (rounds to 1 year). However, the output "last year" is not correct, since it's the year before.

I've addressed this in #765 and the output will now be "1 year ago" (until the 1.5 years threshold is reached, then it will flip to "2 years ago").
